katas/content/single_qubit_gates/amplitude_change/solution.md

```diff
@@ -21,7 +21,7 @@ $$
 
 Recall that when applying a gate, you can tell what its matrix does to the basis states by looking at its columns: the first column of the matrix is the state into which it will transform the $\ket{0}$ state, and the second column is the state into which it will transform the $\ket{1}$ state.
 One of the examples used by the testing harness has $\beta = 0.6, \gamma = 0.8$ and $\alpha = \frac{\pi}{3} = 1.0472$.
```

Comment:
Saying \frac{\pi}{3} = 1.0472 is imprecise. The next sentence mentions 'working to 4 decimal places', but the scope is limited to that sentence only. Should we use '≈'?